### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9273]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Heading levels **should** only increase by one. 

* Side note: There's a typo in the "(VA From 10-10CG)" which should be Form.

## Potential fixes

* The "You’ve successfully submitted your application." heading is currently h3, but is not preceded by an h2. This should be an h2 with with h3 utility styling.

* The "Application for the Program of Comprehensive Assistance for Family Caregivers (VA Form 10-10CG)" is currently an h4, but is not preceded by an h2. This should be an h3 or an h2 with h4 utility styling.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

- Operating System: all
- Browser: all
- Screenreading device: any
- Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers in browser
2. Navigate through the form to the confirmation screen
3. Run an axe audit
4. Verify that the heading levels should increase by one error displays, referencing these headings

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

## Screenshots or Trace Logs

![caregiver-confirmation-headings](https://user-images.githubusercontent.com/57469/82359746-4f420800-99d6-11ea-9a2c-198f3dbefe4f.png)

**Comment by ala-yna** [comment]
@jenstrickland - Can you validate and close? Thank you!

**Comment by jenstrickland** [comment]
@ala-yna There seems to be an issue with the form. I'm not able to get to this screen as there's an issue preventing submitting the form right now. I did check the headings of the page with the error and the heading issue is addressed.

![Screen Shot 2020-06-15 at 12 34 10 PM](https://user-images.githubusercontent.com/57469/84683029-8ee00f00-af04-11ea-9162-923e14f85f2a.png)

**Comment by ala-yna** [comment]
@LinnJS - Since it's a little hard to trigger the successful application page at the moment - can you take a screenshot of the success page so we can validate and close the ticket?

**Comment by jenstrickland** [comment]
@LinnJS — In addition to a screenshot of the success page, would you please include the axe scan showing that there is only 1 heading level issue (for the h4 in the footer)? Thank you!

**Comment by LinnJS** [comment]
@jenstrickland Here is the axe scan showing these bugs being fixed

![image.png](https://images.zenhubusercontent.com/5dcf313b98e0b800015f2353/0b977bc5-0d51-44f2-9e55-f48bf2d80ec5)

Also about the successful submissions it was giving me some trouble at first too. The Veteran must be in MPI/MVI as well as the API must have a mock response already mocked for it to work currently.  I will message you will a user that will work.

**Comment by jenstrickland** [comment]
Validated. Thank you! 🎉
